x-pack/plugins/osquery/server/routes/live_query/get_live_query_results_route.ts

```diff
@@ -88,10 +90,12 @@ export const getLiveQueryResultsRoute = (router: IRouter<DataRequestHandlerConte
                   request.query.page ?? 0,
                   request.query.pageSize ?? 100
                 ),
-                sort: {
-                  direction: request.query.sortOrder ?? 'desc',
-                  field: request.query.sort ?? '@timestamp',
-                },
+                sort: [
+                  {
+                    direction: (request.query.sortOrder as Direction) ?? Direction.desc,
```

Comment:
Not necessary to cast the type here. It should infer it as `"asc" | "desc" | Direction`

Reply:
Not really, it doesn't work  :P 
But, I adjusted the schema to accept direction instead of asc | desc, so let's see if this still works 🤞